react/features/base/react/components/NavigateSectionList.js

```diff
@@ -180,7 +203,9 @@ class NavigateSectionList extends Component<Props> {
             <NavigateSectionListItem
                 item = { item }
                 key = { key }
-                onPress = { this._onPress(url) } />
+                onPress = { url ? this._onPress(url) : undefined }
+                secondaryAction = {
+                    url ? undefined : this._onSecondaryAction(id) } />
```

Comment:
secondaryAction sounds generic, but its use is rather specific, because it depends on the url (see check above) and the render is actually a specific "+" button. We need to address this in favor of making the implementation more generic (instead of just changing the name), because on web we need to show a "Join" button when there's no URL.